dialoptions.go

```diff
@@ -527,10 +527,7 @@ func WithDefaultServiceConfig(s string) DialOption {
 // WithDisableRetry returns a DialOption that disables retries, even if the
 // service config enables them.  This does not impact transparent retries, which
 // will happen automatically if no data is written to the wire or if the RPC is
-// unprocessed by the remote server.
-//
-// Retry support is currently enabled by default, but may be disabled by
-// setting the environment variable "GRPC_GO_RETRY" to "off".
+// unprocessed by the remote server. Retry support is enabled by default.
```

Comment:
Can we omit the last sentence. Maybe its just me, but I feel that it adds some confusion.